lib/positions.js

```diff
@@ -131,6 +136,18 @@ exports.editPosition = async (req, res) => {
     });
 };
 
+exports.deletePosition = async (req, res) => {
+    if (!req.permissions.manage_candidates) {
+        return errors.makeForbiddenError(res, 'You cannot manage positions.');
+    }
+
+    await req.position.update({ deleted: true });
+
+    return res.json({
+        succes: true,
+        data: req.position
```

Comment:
```suggestion
        succes: true
```
I'd do either this, or do an empty body in combination with a 204 status code. The position does no longer exist, so I don't see a point in returning it.

Reply:
it's better then to return

```
{
   success: true,
   message: 'The position was deleted.'
}
```

Reply:
@rinkp we also have a convention for the standardized API format somewhere in the confluence. Basically idea is that every response contains `success` as a boolean, either `data` or `message` if `success: true`, and either `errors` or `message` if `success: false`